config/install/tfa.settings.yml

```diff
@@ -1,6 +1,10 @@
 enabled: 0
+secret_key: ''
```

Comment:
https://github.com/d8-contrib-modules/tfa/issues/43 
I filed an issue though I have no idea about the key module, please update the issue to explain the requirement and steps to be taken to implement it would be good too but not necessary, I can explore it later.